src/main/java/calculator/Operator.java

```diff
@@ -0,0 +1,16 @@
+package calculator;
+
+public enum Operator {
+
+    PLUS('+'), SUBSTRACT('-'), MULTIPLY('*'), DIVIDE('/') ;
+
+
+    private char operator;
+    Operator(char operator){
+        this.operator = operator;
+    }
```

Comment:
연산자 객체 분리를 잘해주셨습니다 👍 
연산자 객체를 별도로 분리함으로써 한 곳에서 연산에 대한 정보와 행위를 모아두어 응집도를 높이고
Operator라는 타입을 부여하여 다룰 수 있도록하여
스스로 상태를 관리하고 해당 상태에 맞게 판단하여 행동할 수 있다고 생각합니다 :)

enum의 장점은 가독성과 값과 행위를 한곳에서 관리한다는 점인 것 같아요.
이 점을 이용해 enum을 활용한다면 값과 메소드가 어떤 관계가 있는지에 대해 더이상 다른 곳을 찾을 필요가 없고
Enum에게 직접 물어보면 되기 때문입니다.

해당 연산에 맞게 행위를 하는 객체를 다형성으로 통해 구현해보시면 좋은 경험이 될 것 같아요!
```suggestion
    PLUS('+', new PlusCalculator())
    , SUBSTRACT('-', new SubtractCalculator())
    , MULTIPLY('*', new MultiplyCalculation())
    , DIVIDE('/', new DivideCalculation());

    private char operator;
    private Calculator calculator;
     
    Operator(char operator, Calculator calculator) {
        this.operator = operator;
        this.calculator = calculator;
    }
```